cmd/babylond/cmd/root.go

```diff
@@ -127,12 +126,8 @@ func NewRootCmd() *cobra.Command {
 	// add keyring to autocli opts
 	autoCliOpts := tempApp.AutoCliOpts()
 	initClientCtx, _ = config.ReadFromClientConfig(initClientCtx)
-	autoCliOpts.Keyring, _ = keyring.NewAutoCLIKeyring(initClientCtx.Keyring)
+	// autoCliOpts.Keyring, _ = keyring.NewAutoCLIKeyring(initClientCtx.Keyring)
```

Comment:
hmm how are those changes connected to IBC ?

Reply:
it updated the autoCliOpts structure in 
![image](https://github.com/user-attachments/assets/104a5a55-51d8-48d3-a2d1-933f082c6455)

Reply:
oh so this whole bit:
```
	autoCliOpts.TxConfigOpts = tx.ConfigOptions{
		EnabledSignModes:           tx.DefaultSignModes,
		TextualCoinMetadataQueryFn: authtxconfig.NewGRPCCoinMetadataQueryFn(initClientCtx),
	}
```
is no longer needed ?

Reply:
yep, there is no option to set this in the autoCliOps